R/run_stress_test.R

```diff
@@ -391,6 +405,11 @@ sector_exposures_file <- function(dir) {
   return(out)
 }
 
+capacity_factor_file <- function(dir) {
```

Comment:
I would suggest to at some point move all filename creation, reading wrappers and processing wrappers out of the run_ file to some more specialised files

Reply:
Defer it to #308